### ISSUE TITLE
inital lambda not being calculated

### ISSUE THREAD

**Issue by briochh** [jtwhite79/pestpp#9]
Previously the intial lambda value for `ies`, if not explicitly set using `++ies-initial_lambda` was being automatically calculated, based on the original phi value.

It appears that this is not happening any more (at least not for me). If I do not specify and inital lamba it appear to be set to 10, which, maybe incidentally, is the inital lambda in the pest control data section.

My pest++ options are:
```
++OVERDUE_GIVEUP_FAC(3.0)
++OVERDUE_RESCHED_FAC(3.1)
++MAX_RUN_FAIL(1)
++ies_localizer(localizer.mat)
++OVERDUE_GIVEUP_MINUTES(360)
++ies_parameter_ensemble(rm_south.par.jcb)
++sweep_output_csv_file(rm_south.out.csv)
++sweep_parameter_csv_file(rm_south.par.csv)
++ies_obs_en(obs.jcb)
++ies_bad_phi(10000000000.0)
```

**Comment by jtwhite79** [comment]
Looking at the code, I don't seen anything obvious as to how this calculation could be borked.  Can you send me your files?

**Comment by jtwhite79** [comment]
this was an easy one -  switched from ``nobs`` to ``nzobs`` for this calculation.  Fixed as of ``4.1.8``
